contracts/escrow.sol

```diff
@@ -162,4 +161,24 @@ contract Escrow is Pausable {
     */
     function() external {
     }
+
+    /**
+     * @dev Rates a transaction
+     * @param _escrowId Id of the escrow
+     * @param _rate rating of the transaction from 1 to 5
+     * @notice Requires contract to not be paused.
+     *         Can only be executed by the buyer
+     *         Transaction must released
+     */
+    function rate_transaction(uint _escrowId, uint _rate) public whenNotPaused {
+        require(_rate >= 1, "Rating needs to be at least 1");
+        require(_rate <= 5, "Rating needs to be at less than or equal to 5");
+
+        EscrowTransaction storage trx = transactions[_escrowId];
+
```

Comment:
what do you think of adding a require(trx.rating == 0) so people can set their rating only once?

Reply:
I think you might want to change your rating, like on Amazon, you can change your mind. 
I agree however that a transaction cannot break or something like an object can.